src/dnspingsender.cpp

```diff
@@ -59,13 +59,13 @@ DnsPingSender::DnsPingSender(const QHostAddress& source, QObject* parent)
 
 DnsPingSender::~DnsPingSender() { MZ_COUNT_DTOR(DnsPingSender); }
 
-void DnsPingSender::start() {
+bool DnsPingSender::start() {
   auto state = m_socket.state();
-  if (state != QAbstractSocket::UnconnectedState) {
+  if (true) {
```

Comment:
Hum, is this a mistake? Doesn't this invalidate the whole rest of this function?